### DISCUSSION TITLE
Fortnite Support

### DISCUSSION THREAD

**Discussion by emirchus** [Heroic-Games-Launcher/HeroicGamesLauncher#213] [Q&A]
Hi all, I wanted to ask about Fortnite support, since the Linux 5.11.x kernel is supposed to support EAC and stuff, plus there was a leak about the native support for the game (https://twitter.com/Lucas7yoshi/status/1361608275905679363)

**Comment by not-a-dev-stein** [comment]
It isn't "supposed to support EAC", the syscall dispatch feature in 5.11 is to have better performance/compatibility with more intrusive DRM, it might eventually be used as a base for anticheat support, but it in itself has nothing to do with anticheat support.
